src/SprykerEco/Zed/Payone/Dependency/Service/PayoneToUtilEncodingServiceBridge.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Dependency\Service;
+
+use Spryker\Service\UtilEncoding\UtilEncodingServiceInterface;
+
+class PayoneToUtilEncodingServiceBridge implements PayoneToUtilEncodingServiceInterface
+{
+    /**
+     * @var \Spryker\Service\UtilEncoding\UtilEncodingServiceInterface
+     */
+    protected UtilEncodingServiceInterface $utilEncodingService;
+
+    /**
+     * @param \Spryker\Service\UtilEncoding\UtilEncodingServiceInterface $utilEncodingService
+     */
+    public function __construct($utilEncodingService)
+    {
+        $this->utilEncodingService = $utilEncodingService;
+    }
+
+    /**
+     * @param string $jsonValue
+     * @param bool $assoc Deprecated: `false` is deprecated, always use `true` for array return.
+     * @param int|null $depth
+     * @param int|null $options
+     *
+     * @return object|array<mixed>|null
+     */
+    public function decodeJson(string $jsonValue, bool $assoc = false, ?int $depth = null, ?int $options = null)
+    {
+        if ($assoc === false) {
```

Comment:
Why do we need it here? remove it pls.

Reply:
We need it in order our old bridges to support/force using newer versions according to our arch sniffer